libpod/define/container.go

```diff
@@ -0,0 +1,8 @@
+package define
+
+// PruneContainerResponse describes the results of a container pruning
+type PruneContainerResponse struct {
```

Comment:
Why in define?

Reply:
Also, for consistency with the rest of libpod, we probably want to return:
[]string, map[string]error, int64, error
(Array of IDs of successes, map of ID to error for failed containers, total space freed, fatal errors that prevented us from pruning any container)

We can convert to this in adapter